app/views/ReportView/components/Pharmacogenomic/columnDefs.ts

```diff
@@ -39,15 +39,11 @@ const columnDefs: ColDef[] = [{
       return `${variant.gene.name} ${variant.cnvState}`;
     }
     if (variantType === 'sv') {
-      return `(${
```

Comment:
here as well

Reply:
Honestly think the original looks clearer, but if autofix decides that it should change ...